.gitignore

```diff
@@ -1,5 +1,6 @@
 # npm assets
 node_modules/
+package-lock.json
 
 # Hugo-generated assets
 /public
```

Comment:
Could you also delete the extra blank line at the end of this file?

Reply:
So that we match the change made on `docsy`?

> <img src="https://user-images.githubusercontent.com/4140793/112370525-e55c5400-8cb3-11eb-84c5-bb7728bada40.png" width=300>

Reply:
Weird. It's there in my local version... lemme try again.

Reply:
sorry, I misread. I can delete it yes.